src/extensions/extension.tsx

```diff
@@ -217,9 +213,10 @@ async function appendInformationToLocalStorage(array, type: RepoType) {
   // This system should make it so themes and extensions are stored concurrently
   for (const repo of array.items) {
     // console.log(repo);
-    const data = (type === "theme")
-      ? await fetchThemeManifest(repo.contents_url, repo.default_branch, repo.stargazers_count)
-      : await fetchExtensionManifest(repo.contents_url, repo.default_branch, repo.stargazers_count);
+    const data =
+      type === "theme"
+        ? await fetchThemeManifest(repo.contents_url, repo.default_branch, repo.stargazers_count)
+        : await fetchExtensionManifest(repo.contents_url, repo.default_branch, repo.stargazers_count);
```

Comment:
Most likely because there is a fixed limit on how long a line should be iirc. I extended it from the default value but that's prob not enough.

Reply:
I mean like this, which has shorter line length because of one less indent on the long lines. 
<img width="823" alt="image" src="https://user-images.githubusercontent.com/1565883/173075967-86f46238-e3ce-47e8-a368-69397b23d79f.png">

Reply:
Ah I see. Will look into it.

Reply:
![image](https://user-images.githubusercontent.com/77577746/173086802-c49345c9-299f-4058-9f09-be4379673d06.png)
I don't think this is configurable tbh, can't find any references anywhere. Probably because it treats the ternary as a separate block of code.